modules/odc-connecting-components.adoc

```diff
@@ -51,21 +54,24 @@ features, see https://access.redhat.com/support/offerings/techpreview/.
 ====
 
 
-
 [NOTE]
 ====
-Currently, only a few specific Operators like the *etcd* and the *PostgresSQL Database* Operator's service instances are bindable.
+Currently, a few specific Operators like the *etcd* and the *PostgresSQL Database* Operator's service instances are bindable.
 ====
 
-To connect the Node.js component using a binding connector:
+You can establish a binding connection with Operator-backed components.
+
+This procedure walks through an example of creating a binding connection between a PostgreSQL Database service and a Node.js application.
 
 .Prerequisite
 * Ensure that you have created and deployed a Node.js application using the *Developer* perspective.
 * Ensure that you have installed the *Service Binding Operator* from *OperatorHub*.
 
 .Procedure
 
-. Install the DB Operator using a backing `OperatorSource`. A backing `OperatorSource` exposes the binding information in secrets, ConfigMaps, status, and spec attributes.
+To create a binding connection with a service that is backed by the PostgreSQL Database Operator, you need to first add the Red Hat provided PostgreSQL Database Operator to the *OperatorHub* using a backing `OperatorSource`, and then install the Operator.
```

Comment:
Is there a reason that this sentence is necessary here? I don't think we normally put a lead-in sentence summarizing the procedure in front of the procedure steps.

If you do think it's necessary, then I would put it above in the sentences before the prerequisites. And also "you need to" -> "you must" and "Red Hat provided" -> "Red Hat-provided"